packages/react/src/components/ListItem/index.stories.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import {
+  withKnobs,
+  select,
+  boolean,
+  text,
+  number,
+} from "@storybook/addon-knobs";
+import { withInfo } from "@storybook/addon-info";
+import { action } from "@storybook/addon-actions";
+import { ICONS, LIST_ITEM_TYPES } from "../../constants/";
+
+import ListItem from "./";
+
+ICONS.None = null;
+
+storiesOf("List Items")
```

Comment:
Same comment as for `InlineAlert` about the stories organization

Reply:
👍